exercises/beer-song/src/test/java/BeerSongTest.java

```diff
@@ -17,39 +17,39 @@ public void setup(){
     public void singFirstVerse() {
         assertEquals("99 bottles of beer on the wall, 99 bottles of beer.\n" +
                         "Take one down and pass it around, 98 bottles of beer on the wall.\n\n",
-                beerSong.verse(99));
```

Comment:
With this change I guess we can remove the `verse` method from the reference implementation? :)

Reply:
I was wondering this - do you think it would be best to just make the `verse` method private and keep it, or move the logic from `verse` to line 20? 🙂

Reply:
Good question. Either would be fine I think, but I would probably keep the `verse` method and make it private 🙂

Reply:
Agreed 🙂